scripts/cmake/MCWAMP.cmake

```diff
@@ -27,7 +28,7 @@ macro(amp_target name )
   target_include_directories(${name} PRIVATE ${MCWAMP_INC_DIR})
   target_include_directories(${name} SYSTEM INTERFACE $<INSTALL_INTERFACE:$<INSTALL_PREFIX>/include>)
   target_compile_options(${name} PUBLIC -hc -fPIC)
-
+ 
```

Comment:
there was a random trailing space added here

Reply:
doh.  ok will remove.  And there is another one at line 71.